### DISCUSSION TITLE
What needs to be finished before sharing Render site with SW3 authors?

### DISCUSSION THREAD

**Discussion by gwijthoff** [Princeton-CDH/startwords#258] [Announcements]
_(This didn't feel right as a GitHub issue, but I wanted to directly link to each of the issues / PRs below, so I figured I'd try GitHub Discussions. LMK if you'd like this somewhere else.)_

Hi @gissoo @rlskoeser — I'm taking stock of what work is still in process, and what needs to be done before I send the SW3 contributors my editorial comments on their articles and the Render site with a draft of how their content will look.

Does this look right to you both? Anything I'm missing?

| Description                      | №                 | Needed Before Sharing w/ Authors?   |
| -------------------------------- | ----------------- | ----------------------------------- |
| Language toggles                 | #242, #226        | Yes, awaiting @gissoo review on #242 |
| Formatting tables                | #257              | Yes                                 |
| Generalizing interlude styles    | #204, #219        | Probably not                        |
| Update shape of lede shadow text | #255, #254        | No                                  |
| Author list page                 | #252, #251, #249  | No                                  |

**Comment by rlskoeser** [comment]
Thanks for documenting this! haven't used discussions before, but seems like a decent choice for this.

The language toggles were awaiting a design revision from Gissoo — she was going to adjust the perspective a bit to make it easier for me to implement in CSS (right now the alignment is slightly off). I know she was working on it but I don't remember ever seeing the updated designs.

However, we could merge the current PR as a provisional implementation, since the functionality is in place and the visuals are fairly close — that way it wouldn't be a blocker for sharing with contributors.

**Comment by gwijthoff** [comment]
Merging the current PR, provisionally, makes sense to me @rlskoeser. It doesn't have to be perfect at this stage, just enough for the authors to understand the functionality and visualize how their articles will be read.
